qt/widgets/common/src/DataSelector.cpp

```diff
@@ -110,14 +105,8 @@ void DataSelector::handleFileInput() {
     return;
   }
 
-  // attempt to load the file
-  if (m_autoLoad) {
-    emit filesAutoLoaded();
-    autoLoadFile(filename);
-  } else {
-    // files were found
-    emit filesFound();
-  }
+  emit filesAutoLoaded();
```

Comment:
Sorry if I'm not following the code and PR comments very well, but I just wanted to double check that this change is what's intended - it looks like the code here previously used to check whether auto load was enabled/requested, but now it looks like it will always perform the auto-load regardless?

Reply:
Thanks for checking. I found out that the `m_autoLoad` option is always set to true, and so I decided to do a small bit of simplification for this class by removing the option entirely. The only place where we might not want to do auto-loading is now in the `isValid` function, and I have provided an option which can be passed to this function directly

Reply:
Ah, great, thanks for explaining, that all sounds good then!